test/java/org/opendatakit/briefcase/model/FormStatusTest.java

```diff
@@ -43,4 +45,14 @@ public void knows_how_to_build_paths_to_assets_inside_the_storage_directory() {
     );
   }
 
+  @Test
+  public void gets_manifest_url_from_remoteform() {
```

Comment:
This test feels like it's not bringing any value. It doesn't help to describe this the class' behavior with a meaningful example. It doesn't helping prevent bugs either because it only asserts about its initial state.

Test coverage is not a goal by itself, nor it shoule be considered a metrics to get a sense of the quality of a codebase.

Code, on the other hand, is inventory and has inherent cost. That's why I would suggest not writing this test.